src/quo/components/community/channel_action.cljs

```diff
@@ -0,0 +1,39 @@
+(ns quo.components.community.channel-action
```

Comment:
can we move this to a view file.
`quo.components.community.channel-action.view`